### PR TITLE
Zoom reset

### PR DESCRIPTION
**Pull request by raveit65**
This will add zoom-reset function, please test.

### CODE REVIEW

**Comment by sc0w** [comment]
here the "reset zoom" goes to 85 % , the expected is 100 % ?

**Comment by raveit65** [comment]
I think the default was always 85 % or not?

**Comment by raveit65** [comment]
Btw. i don't want to add this deprecated GtkScreen stuff again https://github.com/linuxmint/xreader/commit/ddd6c31b10a630a88bf171072bc75fcd6e420007

**Comment by sc0w** [comment]
I said it, because I see in  **previewer/ev-previewer-window.c**

```
+        { "ViewZoomReset", "zoom-original", NULL, "<control>0",
+          N_("Reset zoom to 100\%"),
+          G_CALLBACK (ev_previewer_window_zoom_reset) },
```

**Comment by raveit65** [comment]
I will rename that.

**Comment by sc0w** [comment]
I only see the build warning:

```
ev-window.c:6681:2: warning: ‘gtk_action_group_get_action’ is deprecated [-Wdeprecated-declarations]
  action = gtk_action_group_get_action (action_group, "ViewZoomReset");
  ^~~~~~
```

but the same function is used in more points, and it can be fixed at same time in the future

**Comment by sc0w** [comment]
I suggest these changes, and the "zoom reset" goes to 100%, the expected for me

```
diff --git a/libview/ev-view.c b/libview/ev-view.c
index 9414336..334540c 100644
--- a/libview/ev-view.c
+++ b/libview/ev-view.c
@@ -7092,7 +7092,7 @@ void
 ev_view_zoom_reset (EvView *view)
 {
 	g_return_if_fail (view->sizing_mode == EV_SIZING_FREE);
-	ev_document_model_set_scale (view->model, 1.0);
+	ev_document_model_set_scale (view->model, 1.2);
 }
 
 static double
```

**Comment by raveit65** [comment]
Thanks for the review and help.
I added your suggestion and rebased the PR.

**Comment by sc0w** [comment]
btw, I noticed the zoom doesn't work with _.epub_ files with 1.20 and this PR

**Comment by raveit65** [comment]
If you add *zoom-in*, *zoom-out* and *zoom-reset* buttons to toolbar you can zoom in epubs.
But the behaviour is complete weird and different to pdfs.
Also the buttons in view menu are working here.
But the *view combobox* in toolbar is broken.

File: shell/ev-window.c [hunk 1]
```diff
@@ -6446,7 +6446,7 @@ static const GtkToggleActionEntry toggle_entries[] = {
         { "ViewFitWidth", EV_STOCK_ZOOM_WIDTH, N_("Fit _Width"), "w",
           N_("Make the current document fill the window width"),
           G_CALLBACK (ev_window_cmd_view_fit_width) },
-	{ "ViewInvertedColors", EV_STOCK_INVERTED_COLORS, N_("_Inverted Colors"), "<control>I",
+	{ "ViewInvertedColors", EV_STOCK_INVERTED_COLORS, N_("Inverted _Colors"), "<control>I",
```
Comment (monsta) [inline shell/ev-window.c]: Oops, now it conflicts with "Continuous" :slightly_smiling_face:

**Comment by raveit65** [comment]
C for "Continuous" isn't a menu accel.

**Pull request merged**